apps/test/solutions/eval/examples.js

```diff
@@ -194,6 +197,51 @@ module.exports = {
       xml: '<xml>' +
         solutionBlocks +
         '</xml>'
+    },
+
+    {
+      description: "example running hides solution and displays result",
+      expected: {
+        result: true,
+        testResult: TestResults.ALL_PASS
+      },
+      customValidator: function (assert) {
+        var answerElement = document.getElementById('answer');
+        var callElement = document.getElementById('test-call');
+        var resultElement = document.getElementById('test-result');
+        var triangleBlock = Blockly.mainBlockSpace
+            .findFunctionExamples("green-triangle")[0];
+        var contractEditor = Blockly.contractEditor;
+
+        // Test result hidden before run
+        assert.equal(answerElement.style.display, 'block');
+        assert.equal(callElement.style.display, 'none');
+        assert.equal(resultElement.style.display, 'none');
+
+        // Run test
+        var resultBeforeTest = Eval.result;
```

Comment:
Without this, the "reset" code in the test handler would clear the Eval.result and for some reason the test would fail with:

```
  1) Level tests ../solutions/eval/examples.js example running hides solution and displays result:
     timeout of 10000ms exceeded. Ensure the done() callback is being called in this test.
```